spec/models/piece_spec.rb

```diff
@@ -15,4 +15,16 @@
     queen = FactoryBot.create(:piece, x_position: 3, y_position: 0, color: :white, type: :queen, game: game)
     expect(queen.obstructed?(3, 3)).to eq false
   end
+
+  it 'will check if a king\'s move is valid' do
+    game = FactoryBot.create(:game)
+    king = FactoryBot.create(:king, x_position: 4, y_position: 0, color: :white, type: :king, game: game)
+    expect(king.valid_move?(4, 1)).to eq true
+    king = FactoryBot.create(:king, x_position: 4, y_position: 0, color: :white, type: :king, game: game)
+    expect(king.valid_move?(4, 3)).to eq false
+    king = FactoryBot.create(:king, x_position: 4, y_position: 0, color: :white, type: :king, game: game)
+    expect(king.valid_move?(4, 0)).to eq true
+    king = FactoryBot.create(:king, x_position: 4, y_position: 3, color: :white, type: :king, game: game)
+    expect(king.valid_move?(5, 0)).to eq false
+  end
```

Comment:
Awesome tests!

You can move these into a new file `king_spec.rb` We'll have more type specific tests, so it's good to split them out.

Another best practice is splitting the assertions out into separate `it blocks`. See here: http://www.betterspecs.org/#single

I'd split out each
```Ruby
king = FactoryBot.create(:king, x_position: 4, y_position: 3, color: :white, type: :king, game: game)
expect(king.valid_move?(5, 0)).to eq false
```
Into it's own block.